internal/driver/interactive.go

```diff
@@ -267,6 +272,9 @@ func parseCommandLine(input []string) ([]string, variables, error) {
 		}
 	}
 	if c == nil {
+		if v := pprofVariables[name]; v != nil {
+			return nil, nil, fmt.Errorf(`did you mean: %s?`, name+"="+args[0])
```

Comment:
Nit: Same as above.
Also nit: I don't think back ticks have added benefit here, so I'd reccomend quotes,
So, following is more idiomatic (assuming args is a slice of strings):
```
fmt.Errorf("did you mean: %s=%s", name, args[0])
```

Reply:
Done.